### DISCUSSION TITLE
softether vpncmd "OpenVpnEnable" Command not found

### DISCUSSION THREAD

**Discussion by shakibamoshiri** [SoftEtherVPN/SoftEtherVPN#1882] [Q&A] [answered]
How to enable OpenVPN on version 5 ?
vpncmd seems not to have `OpenVpnEnable` command

**Comment by chipitsine** [comment, marked as answer]
we enable OpenVPN from command line during testing https://github.com/SoftEtherVPN/SoftEtherVPN/blob/master/.ci/start-se-openvpn.sh#L36
